docs/en/advanced/nn_module_utilities.md

```diff
@@ -0,0 +1,73 @@
+# Neural Network Module: System Integration
```

Comment:
FYI, note the rename. It isn't so much utilities, as "broader integration into PX4". Please read the rest of this.